lms/models/lti_user.py

```diff
@@ -59,6 +66,15 @@ def from_auth_params(application_instance, lti_core_schema):
             email=lti_core_schema["lis_person_contact_email_primary"],
         )
 
+    @staticmethod
+    def unserialize(request, **kwargs: dict):
```

Comment:
LTIUser gets serialized/unserialized as part of different authentication schemas.

I reckon it makes sense to make that explicit.

Not sure about the "unserialize" name thought, we do have `serialize` below

Reply:
I think `deserialize` would be the norm?